src/pages/_borrow/BorrowModal/BorrowStep.tsx

```diff
@@ -48,11 +50,10 @@ export const BorrowModal = ({ modalId, onClose, currency, totalSupplied }: Borro
   const healthFactor =
     loanAmountCents && loanAmountCents > 0n ? Number(collateralAmountCents) / Number(loanAmountCents) : 0;
 
-  // The modal is impossible to open without collateral balance.
-  if (!collateralBalance) return null;
+  // TODO: get this from the contract.
+  const interestRate = '7.5%';
```

Comment:
I might remember wrong, but I think there's a getter for this already.

Reply:
Yeah there is, but I think connecting that is outside this ticket's scope, as this was just refactored from the previous borrow modal. I can connect that getter as the next thing tho!